src/crewai/utilities/events/base_events.py

```diff
@@ -3,12 +3,26 @@
 
 from pydantic import BaseModel, Field
 
+from crewai.flow.state_utils import Serializable, to_serializable
```

Comment:
In a follow up PR, I'll move this package elsewhere. It started to be used for flows, but we're using it in other places as well.